tests/conftest.py

```diff
@@ -6,13 +6,30 @@
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
 import pytest
+from pytest import fixture
 
 import pycds
 from pycds import Network, Contact, Station, History, Variable
 
 def pytest_runtest_setup():
     logging.basicConfig(level=logging.DEBUG, stream=sys.stdout)
 
+@fixture(scope='module')
```

Comment:
Ah good. I saw this in the test files in an earlier commit, and was going to suggest that you added the fixture to conftest.py. But you got it! Great.